pkgs/os-specific/linux/picoprobe-udev-rules/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchurl }:
+
+## Usage
+# In NixOS, simply add this package to services.udev.packages:
+#   services.udev.packages = [ pkgs.picoprobe-udev-rules ];
+
+stdenv.mkDerivation rec {
+  pname = "picoprobe-udev-rules";
+  version = "0.0.1";
+
+  src = fetchurl {
+    url = "https://probe.rs/files/69-probe-rs.rules";
```

Comment:
Aren't there some *permanent* url whose content won't change over time?

Reply:
I don't think so, at least not in the official repository.

Reply:
Found it here https://github.com/probe-rs/webpage/blob/master/static/files/69-probe-rs.rules

Reply:
Oh, I'll port it to the latest commit from that repo, thanks!

Reply:
And just set the version to `unstable-YYYY-MM-DD`.